mne/tests/test_epochs.py

```diff
@@ -506,7 +528,7 @@ def test_epoch_multi_ids():
     epochs = Epochs(raw, events, {'a/b/a': 1, 'a/b/b': 2, 'a/c': 3,
                                   'b/d': 4, 'a_b': 5},
                     tmin, tmax, picks=picks, preload=False, add_eeg_ref=False)
-    epochs_regular = epochs[['a', 'b']]
+    epochs_regular = epochs['a/b']
```

Comment:
We should probably have a test for order invariance here; what I mean is, epochs["a/b"] should be checked to also select "b/a". The way the code is right now, this is trivially and inherently done, but it might still be worth testing explicitly.